### PR TITLE
Updates on LD test files

### PR DESCRIPTION
**Pull request by fattaneh88**
I generalized the solution based on the two main input parameters, and based on that I changed the LD code.

### CODE REVIEW

**Comment by rbharath** [comment]
@fattaneh88 Are there some missing files? Seeing only a 1 line change here

**Comment by fattaneh88** [comment]
No it is more than a line. Let me check LD file, probably, it is not the most recent version!

**Comment by fattaneh88** [comment]
LD is updated now.

File: test/LD.c [hunk 1]
```diff
@@ -4,15 +4,21 @@
 #include <time.h>
 #include "../include/justGarble.h"
 
-#define N_allels 10000
-//N_AB
+//two main parameters of LD test
+#define M_allels 10000
```
Comment (rbharath) [inline test/LD.c:8]: Typo? Should this be `M_alleles`?

File: test/LD.c [hunk 2]
```diff
@@ -26,8 +32,8 @@ int main(int argc, char **argv) {
     
     //Set up circuit parameters
     int q, r;
```
Comment (rbharath) [inline test/LD.c:34]: What are `q` and `r`? Could you add a comment explaining?

File: test/LD.c [hunk 3]
```diff
@@ -4,15 +4,21 @@
 #include <time.h>
 #include "../include/justGarble.h"
 
-#define N_allels 10000
-//N_AB
+//two main parameters of LD test
+#define M_allels 10000
+
+//N_AB, N_Ab, N_aB, N_ab
 #define Size_I 8
```
Comment (rbharath) [inline test/LD.c:11]: Can you add a comment explaining what `Size_I` is?

**Review by rbharath — CHANGES_REQUESTED** [review]
Looks good for the most part. I have a couple requests for more comments I've indicated. Should be good to merge in after

File: test/LD.c (hunk 1 above)
Comment (fattaneh88) [reply test/LD.c:8]: Based on the recent updates on the documentation (experimental results section), I changed N_allels to M_allels.

File: test/LD.c (hunk 2 above)
Comment (fattaneh88) [reply test/LD.c:34]: I added more comments

File: test/LD.c (hunk 3 above)
Comment (fattaneh88) [reply test/LD.c:11]: I add a comment about Size_I

**Pull request merged**